<!DOCTYPE html>
<!--[if lt IE 8 ]><html class="no-js ie ie7" lang="en"> <![endif]-->
<!--[if IE 8 ]><html class="no-js ie ie8" lang="en"> <![endif]-->
<!--[if (gte IE 8)|!(IE)]><!--><html class="no-js" lang="en"> <!--<![endif]-->
<head>

  <!--- Basic Page Needs
  ================================================== -->
  <meta charset="utf-8">
  <title>Jaime's website</title>
  <meta name="description" content="">
  <meta name="author" content="">

  <!-- Mobile Specific Metas
  ================================================== -->
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

  <!-- CSS
  ================================================== -->
  <link rel="stylesheet" href="css/default.css">
  <link rel="stylesheet" href="css/layout.css">
  <link rel="stylesheet" href="css/media-queries.css">
  <link rel="stylesheet" href="css/magnific-popup.css">
  
  <!-- Script
  ================================================== -->
  <script src="js/modernizr.js"></script>

  <!-- Favicons
  ================================================== -->
  <link rel="shortcut icon" href="haab-pax.png" >

  <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-108396069-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-108396069-1');
</script>


</head>

<style type='text/css'>

    .common {
  float: left;
  position: relative;
  display: block;
  clear: both;
  }
  
  .title_header {
    color: white;
  }
  
  .paper_img {
    height: 140px;
    width: 100px;
  }

</style>

<body>

  <!-- Header
  ================================================== -->
  <header id="home">

    <nav id="nav-wrap">

      <a class="mobile-btn" href="#nav-wrap" title="Show navigation">Show navigation</a>
      <a class="mobile-btn" href="#" title="Hide navigation">Hide navigation</a>

      <ul id="nav" class="nav">
        <li class="current"><a class="smoothscroll" href="#home">Home</a></li>
        <li class="current"><a class="smoothscroll" href="#about">About</a></li>
	<li class="current"><a class="smoothscroll" href="#resume">Resume</a></li>
        <li class="current"><a class="smoothscroll" href="#research">Research</a></li>
        <li class="current"><a class="smoothscroll" href="#publications">Publications</a></li>
      <li class="current"><a class="smoothscroll" href="#parallel_dimensions">Parallel dimensions</a></li>
      </ul> <!-- end #nav -->

    </nav> <!-- end #nav-wrap -->

    <div class="row banner">
      <div class="banner-text">

	<h2 style="color:white"><span>Jaime A Castro-Mondrag√≥n's webiste</span></h2>
        <hr/>
        
        <h3><p align="justify">I am a computational biologist from üá≤üáΩ living in üá≥üá¥ with +10 years of experience in research, sequence analysis, methods development, and data visualisation. I have worked in both the academic and private sector. My main expertise is the development of algorithms to analyze biological data and apply these methods in a wider range of biomedical fields such as vaccine design or gene expression. I am currently working as Bioinformatician in the precision medicine group <a href="https://www.ahus.no/fag-og-forskning/forskning-og-innovasjon/forskningsgrupper/medisinsk-divisjon/epigen" target="_blank">(EpiGen)</a> at the Akersus Universitetssykehus (Ahus) in Oslo.
        </p>
        <ul class="social">
	  <li><a href="#about"><i class="fa fa-user"></i></a></li>
          <li><a href="#resume"><i class="fa fa-folder-open"></i></a></li>
          <li><a href="#research"><i class="fa fa-flask"></i></a></li>
          <li><a href="#publications"><i class="fa fa-book"></i></a></li>
          <li><a href="#parallel_dimensions"><i class="fa fa-plus-circle"></i></a></li>
        </ul>
      </div>
    </div>

    <p class="scrolldown">
      <a class="smoothscroll" href="#about"><i class="icon-down-circle"></i></a>
    </p>

  </header> <!-- Header End -->
   
  <!-- About Section
  ================================================== -->
  <section id="about">
    
    <div class="row">
      
      <div class="three columns">
	
        <img class="profile-pic"  src="images/Jcastro2.jpg" alt="" />
	
      </div>
      
      <div class="twelve columns main-col">
	
        <h2>Summary</h2>

<!--	
        <p align="justify"> I obtained a bachelor degree in <a target="_blank" href="http://www.lcg.unam.mx/"><b>Genomic Sciences</b></a> from UNAM (Universidad Nacional Autonoma de Mexico) in Cuernavaca, Mexico. After that I moved to France where I obtained a master degree in Systems Biology from the <a target="_blank" href="https://www.psl.eu/en/education/master-s-degree-life-sciences">IMaLiS program</a>  at the ENS (Ecole normale Superieure) in Paris, France. Also in France I obtained a PhD degree from the Aix-Marseille Universite in Marseille in 2017. From 2017 to 2022 I worked as a post-doctoral researcher at <a target="_blank" href="https://mathelierlab.com/">Anthony Mathelier's lab</b></a> in Oslo, Norway. Currently I work as researcher at <a href="https://nykode.com/" target="_blank">Nykode Therapeutics AS</a> in Oslo.</p>
-->
	      <p align="justify"> I am a Computational Biologist with a PhD degree in Bioinformatics (Aix-Marseille Universite, France, 2017). I have +10 years of experience as researcher in the academic and private sector.
	                          My main expertise is in the analysis of biological sequences (DNA, RNA, proteins) and the development of user-friendly bioinformatics tools with a strong emphasis in data visualization.
	                          I have experience in multi-omics integration, machine learning, and data visualization. Also as reviewer in scientifc journals and training students.
	                          I am currently working as a Bioinformatician in the <a href="https://www.ahus.no/fag-og-forskning/forskning-og-innovasjon/forskningsgrupper/medisinsk-divisjon/#epigen" target="_blank">precision medicine (EpiGen) group</a> at Akershus Universitetssykehus (Ahus).</p>
	      </p>
    
        <div class="row">
	  
          <div class="columns contact-details">
	    
            <h2>Contact Details</h2>
	    
            <p class="address" style="width: 1500px;">
	      <span>Jaime A Castro-Mondragon</span><br>
	      <span> EpiGen, Medisinsk divisjon<br>
          Akershus Universitetssykehus HF<br>
          Postboks 1000<br>
          1478 L√∏renskog, Norway<br><br>
    
          <span><b style="color:#f0f0f0">E-mail:</b></span> j.a.castro.mondragon@gmail.com<br><br>


        <ul style="list-style-type:circle;">
          <li>ü¶ã Bluesky: <a target="_blank" href="https://bsky.app/profile/jaimicore.bsky.social">@jaimicore</a></li>
          <li>Web of science (prev. Publons): <a target="_blank" href="https://www.webofscience.com/wos/author/record/P-6297-2019">P-6297-2019</a></li>
		      <li>ORCID: <a target="_blank" href="https://orcid.org/0000-0003-4069-357X">0000-0003-4069-357X</a></li>
		      <li>Github: <a target="_blank" href="https://github.com/jaimicore">jaimicore</a></li>
		      <li>Bitbucket: <a target="_blank" href="https://bitbucket.org/jaimicore/">jaimicore</a></li>
          <li>Google scholar: <a target="_blank" href="https://scholar.google.fr/citations?user=pcevKk0AAAAJ&hl=en">pcevKk0AAAAJ</a></li><br>
          </ul>
              </span><br>
              </p>
          </div>
        

<!--		Mastodon : <a target="_blank" href="https://genomic.social/@jaimicore">@jaimicore</a><br> -->

	  
          <div class="columns download">
            <p>
              <a target="_blank" href="Doc/Jaime_Castro_CV.pdf" class="button"><i class="fa fa-download"></i>Download CV</a>
            </p>
          </div>
	  
        </div> <!-- end row -->
	
      </div> <!-- end .main-col -->
      
    </div>
    
  </section> <!-- About Section End-->
  
  
  <!-- Resume Section
  ================================================== -->
  <section id="resume"> 



            <!-- Education
    ----------------------------------------------- -->
    <div class="row skill"> <!-- Skill :  no line at the bottom -->
      
      <div class="three columns header-col">
        <h1><span>Work experience</span></h1>
      </div>
      
      <div class="nine columns main-col"> 

	<div class="row item">
  
	  <div class="twelve columns">

	    <h3>Bioinformatician </h3>
	    <p class="info"><em class="date">2025 - Present</em><span>&bull;</span> Oslo, Norway</p>
	    
	    <p>
	      <a target="_blank" href="https://www.ahus.no/fag-og-forskning/forskning-og-innovasjon/forskningsgrupper/medisinsk-divisjon/epigen">Precision Medicine (EpiGen) group</a>, Oslo, Norway.  </br> </br>
	      Development of reports to be shared with molecular biologist, pathologists and oncologists with the summary of the mutational status of cancer samples.
	      Development, and maintenance of bioinformatic workflows (prediction of single nucleotide variants, structural variants, copy number alterations and gene fusions).
	     
	    </p>
	    
	    <h3>Bioinformatician (Scientist II)</h3>
	    <p class="info"><em class="date">2022 - 2025</em><span>&bull;</span> Oslo, Norway</p>
	    
	    <p>
	      <a target="_blank" href="https://nykode.com/">Nykode Therapeutics AS</a>, Oslo, Norway.  </br> </br>
	      Design and analysis of RNA‚Äëseq experiments to highlight transcriptomic differences in mouse cells tested with different vaccine candidates.
	      Development, establishment, and maintenance of bioinformatic workflows (prediction of MHC‚Äëbinding peptides, protease cleavage sites, RNA
secondary structure).
        Train ML models to highlight protein features explaining experimental observations.
	     
	    </p>
	    
	    
	    
	  </div>
	  
	</div> <!-- item end -->
	
	<div class="row item">
	  
          <div class="twelve columns">
	    
            <h3>Post-doctoral researcher</h3>
            <p class="info"><em class="date">2017 - 2022 </em><span>&bull;</span> Oslo, Norway</p>
	    
            <p align="justify">
              <a href="https://www.med.uio.no/ncmm/english/">Norwegian Centre for Molecular Biosciences and Medicine</a>, Oslo, Norway.<br>

	      <strong>Supervisor</strong>: <a target="_blank" href="https://mathelierlab.com/">Anthony Mathelier</a>.</br><br>

          Multi-omics integration by associating somatic mutations with changes in gene expression (RNA-seq, CNA, SNVs) in multiple cancer types. Also, I worked in the update of the <a target="_blank" href="https://jaspar.elixir.no/">JASPAR database</a>  as a curator and developer.  I peer-reviewed publicationf for journals and trained students. As a side project, I continued working as part of the <a target="_blank" href="http://rsat.france-bioinformatique.fr/rsat/RSAT_portal.html">RSAT team</a> by maintaining the tools I developed during my PhD and updating the motif databases.
	    </p>
	  </div>
	</div> <!-- item end -->

      </div> <!-- main-col end -->
      
    </div> <!-- End Education -->
  
   

    <!-- Education
    ----------------------------------------------- -->
    <div class="row skill"> <!-- Skill :  no line at the bottom -->
      
      <div class="three columns header-col">
        <h1><span>Education</span></h1>
      </div>
      
      <div class="nine columns main-col"> 

	<div class="row item">
  
	  <div class="twelve columns">
	    
	    <h3>PhD in Bioinformatics</h3>
	    <p class="info"><em class="date">2014 - 2017</em><span>&bull;</span> Marseille, France</p>
	    
	    <p>
	      <a target="_blank" href="http://www.univ-amu.fr/">Aix-Marseille Universite</a>, France.</br>
	      <strong>Supervisor</strong>: <a target="_blank" href="https://orcid.org/0000-0002-8799-8584">Jacques van Helden</a>.</br>
	      <strong>Goals</strong>:</br>(1) Development of a tool to cluster, align and reduce redundance of Transcription Factors Binding Motifs.</br>(2) Development of a tool to detect positionally enriched/depleted motifs.</br>
	      <strong>Title</strong>:<a target="_blank" href="Doc/Jaime_Castro_PhD_thesis.pdf"> Development of bioinformatics methods for the analysis of large collections of transcription factor binding motifs: positional motif enrichment and motif clustering</a>. 
	    </p>
	    
	  </div>
	  
	</div> <!-- item end -->
	
	<div class="row item">
	  
          <div class="twelve columns">
	    
            <h3>Master degree in Systems Biology</h3>
            <p class="info"><em class="date">2014</em><span>&bull;</span> Paris, France</p>
	    
            <p>
              <a href="http://www.biologie.ens.fr/depbio/?lang=fr">√âcole Normale Sup√©rieure</a> Paris, France.
	      Courses taken included: Bioinformatics, Genomics, Logical Modelling, Microscopy, Evolution, Molecular Biology.</br>
	      <strong>Master thesis</strong> at Aix-Marseille Universit√© TAGC in the <a href="https://tagc.univ-amu.fr/">TAGC</a> Laboratory in Marseille, France.</br>
	      <strong>Supervisor</strong>: <a target="_blank" href="https://orcid.org/0000-0002-8799-8584">Jacques van Helden</a>.</br>
	      <strong>Title</strong>:<a target="_blank" href="Doc/M2_report_Jaime_Castro.pdf"> matrix-clustering: a novel tool to cluster and align Transcription Factor binding motifs</a>. 
	    </p>
	  </div>
	</div> <!-- item end -->


	<div class="row item">
  
	  <div class="twelve columns">
	    
	    
	    <h3>Bachelor in Genomic Sciences</h3>
	    <p class="info"> <em class="date">2009 - 2013</em><span>&bull;</span> Cuernavaca, Mexico</p>
	    <p>
	      I did a research internship at the  <a target="_blank" href="http://www.ccg.unam.mx/en/ComputationalGenomics"><b>Computational Genomics Lab</b></a>, Centro de Ciencias Gen√≥micas, UNAM, Cuernavaca, Mexico.<br>
	      Supervisors : <a target="_blank" href="https://www.ccg.unam.mx/en/pedro-julio-collado-vides/">Julio Collado-Vides</a> & <a target="_blank" href="https://liigh.unam.mx/amedina/">Alejandra Medina-Rivera</a>.<br/>
	      <b>"Prediction of novel regulons based on several properties of the regulatory network."</b> Based in that most knowns TFs in E. coli K12 are autorregulated, we searched the motifs for the unknown TFs.
	      
	    </p>
	  </div>
	  
	</div> <!-- item end -->
	
      </div> <!-- main-col end -->
      
    </div> <!-- End Education -->

  </section> <!-- Resume Section End-->




  <!-- Research Section
  ================================================== -->
   <section id="research">
  <h2><a href="#research">.</a></h2>


    <div class="row">
      
      <div class="two columns header-col">
	<h1><span></span></h1>
	
      </div>
      
      <div class="ten columns flex-container main-col">
        <h2><a>Research</a></h2>
        
        <div class="flexslider">
          
          <p style="text-align:justify; display:block; float:left;"><br> My projects are mainly focused on Bioinformatics of Transcriptional regulation (as user and developer), mainly in algorithms related to Transcription Factor (TF) Binding Motifs and prediction TF Binding Sites, although Ihave experience in RNA-seq, cancer genomics, miRNA biology, and protease cleavage prediction. I'm an active developer of <a href="http://rsat.france-bioinformatique.fr/rsat/RSAT_portal.html" target="_blank">RSAT</a> which is a collection of tools to analize Cis-Regulatory sequences and I am also involved in the <a href="https://jaspar.elixir.no/" target="_blank">JASPAR</a> database, as curator and processing data for the upcoming release.</p>
	  
          

              <hr>
              <h2 class="title_header">Current Projects</h2>
              <ol>
                
                These are the ongoing projects where I contribute.<br><br>
                
               <!-- <li> My main project for the moment is the study of mutations in TFBSs associated with trans-effect on gene expression. We associated TFBSs with protein-coding and miRNA genes and then assesed the likely association of observing TFBS mutations with changes in expression in the gene networks. The methodology is based on a bayesian network and we applied it to 7 different cancer types from TCGA. Check out the <a href="https://www.biorxiv.org/content/10.1101/2020.06.25.170738v1" target="_blank">preprint</a> of this project. <br><br>
                <img class="common" width="600" height="125" class="pull-left" alt="sample-image" src="images/Dysmir_pic.png">
                <br><br>
                <hr>
              </li> -->
              
              
              <li> I am part of the <a href="https://jaspar.elixir.no/" target="_blank">JASPAR</a> consortium since 2017. With other colleagues, I lead the curation and development of JASPAR during the 2020 and 2022 releases. My contributions were to develop a workflow (using <a href="https://snakemake.readthedocs.io/en/stable/" target="_blank">Snakemake</a>) to ease the motif curation. This workflow performs the motif discovery and the ChIP-seq centrality enrichment score calculation; I am also behind the motif curation, and the visualization of the <a href="https://jaspar.elixir.no//matrix-clusters/" target="_blank">TF motif clusters</a>.<br><br>
                <img class="common" width="300" height="100" class="pull-left" alt="sample-image" src="images/jaspar_logo_2018.png">
                <p class="common"> <br> </p>
                <hr>
              </li>
              
              
               <li> I am an active developer of the Regulatory Sequences Analysis Tools <a href="https://rsat.france-bioinformatique.fr/teaching/RSAT_portal.html" target="_blank">RSAT</a> since 2013. RSAT is a modular software suite for the analysis of cis-regulatory elements in genome sequences (ChIP-seq, ATAC-seq). <br><br>
                <img class="common" width="300" height="100" class="pull-left" alt="sample-image" src="images/RSAT_logo.jpg">
                <p class="common"> <br>We are working in a new release of RSAT with new programs. </p>
                <hr>
              </li>
                
                
                <li><a href="https://rsat.france-bioinformatique.fr/teaching/RSAT_portal.html" target="_blank">RSAT</a> <i>matrix-clustering</i>: a tool to cluster, align and create non-redundant collections of Transcription Factor Binding Motifs. <br><br>Example <a href="./Oct4_example_D3_radial_tree.html" target="_blank"> here</a>.<br><br>
                <img class="common" width="500" height="300" class="pull-left" alt="sample-image" src="images/ECCB_2016_JCastro.jpg">
                <p class="common"><br>This tool can merge different files containing TF binding motifs (i.e. motifs dicovered with different tools) and manually or automatically reduce the redundancy.
                <br><br>I recently extended the algorithm to generate radial (circular) tress including link to databases, some examples can be found on the <a href="https://jaspar.elixir.no/matrix-clusters/" target="_blank">JASPAR website</a>.</p>

                <img class="common" width="500" height="300" class="pull-left" alt="sample-image" src="images/matrix-clustering_jaspar.png">
                
                <p class="common">
                <br><br>The <i>matrix-clustering</i> paper can be found on this <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx314" target="_blank">link</a>.
                <br><br>I am actively maintaining this tool, send me an email or <a href="https://github.com/jaimicore/matrix-clustering_stand-alone/issues" target="_blank">open an issue</a>  in case you want to report a problem. 
                <br><br>
                <br><br> <a href="https://github.com/jaimicore/matrix-clustering_stand-alone" target="_blank">In this repository</a> you can find a faster and stand-alone version (functional but still under active development).
                </p>
                <hr>
              </li>
              
 <!--             <li><a href="http://teaching.rsat.eu/" target="_blank">RSAT</a> <i>position-scan</i>: a tool to detect Transcription Factors positionally biased in sequences (e.g. centered around ChIP-seq peak summits, located near TSS). <br><br>
                <img class="common" width="500" height="300" class="pull-left" alt="sample-image" src="images/Position_scan.png">
                <p class="common"><br>This tool separates the type of positional profiles which allows to identify enrichment or depletion of TF binding sites at certain positions of a set of sequences with the same length.</p>
                <br><br> We used this software to detect TF binding motif enriched at human promoters acting as enhancer (<a href="https://www.nature.com/articles/ng.3884" target="_blank">Epromoters</a>) on two cell lines, see supp figures.
                <hr>
              </li>
              
              <li><a href="http://teaching.rsat.eu/" target="_blank">RSAT</a> <i>matrix-enrichment</i>: a tool to detect gloablly enriched or depleted Transcription Factors in a set of sequences. <br><br>
                <img class="common" width="300" height="500" class="pull-left" alt="sample-image" src="images/Matrix_enrichment.png">
                <p class="common"><br>The TF enrichment can be measured simultaneously in several set of sequences.</p>
                <hr>
              </li>
              
              <li>Using the tool <a href="http://teaching.rsat.eu/" target="_blank">RSAT</a> <i>matrix-clustering</i> I have merged several complete databases with motifs from different taxa (vertebrates, plants and insects) in order to create automatically non-redundant taxon-wise motifs collection. <br><br>
                <img class="common" width="700" height="500" class="pull-left" alt="sample-image" src="images/Non_redundant_databases.png">
                <p class="common"><br>This databases and other examples ara available in this <a href="http://pedagogix-tagc.univ-mrs.fr/rsat/data/published_data/Castro_2016_matrix-clustering/" target="_blank">link</a>. In the future I will include the databases for worms, fungi and bacteria.</p>
                <hr>
              </li>
-->              
            </ol>
              
             <!-- slide ends -->
           

<!-- More slides, if required -->
	    
            <!-- <li> -->
            <!--   <p> <a href="#"><img width="80" height="80" class="pull-left" alt="sample-image" src="images/butterfly1.jpg"></a>I also have a strong interest for the <span>evo-devo field</span>, in particular the <span>evolution of Hox and ParaHox</span> protein sequences accross metazoans. My PhD thesis was focused on the Hox genes. -->
                
            <!--   </p> -->
	      
            <!-- </li> <\!-- slide ends -\-> -->
            
            <!-- <li><a href="#"><img width="80" height="80" class="pull-left" alt="sample-image" src="images/atgc.jpg"></a> -->
            <!--   <p> Among other competences, I analyze <span>next-generation sequencing data</span>. Programs to analyze these data are changing in parallel with the fast improvements of sequencing technologies, making the work of a bioinformatician very dynamic ! -->
	    <!-- 	<br/>On a technical level, I am interested by the technology of <span>Web services</span>, and the development of bioinformatic workflows that take advantage of the SOAP standard. I have contributed to the development of the Web Service access of <a href="http://rsat.scmbb.ulb.ac.be/rsat/" target="_blank">RSAT</a>. </p> -->
	      
            <!-- </li> <\!-- slide ends -\-> -->
	    
	  
        </div> <!-- div.flexslider ends -->
	
	
      </div> <!-- div.flex-container ends -->	     
      
    </div> <!-- Row End-->
    
    
  </section> <!-- Researchd-->
  
  
     <!-- Contact Section
   ================================================== -->
  <section id="publications" >
    <h2><a href="#publications">.</a></h2>
    
           <div class="row">

            <div class="eleven columns header-col">

               <h1><span></span></h1>

            </div>

            <div class="eleven columns">
				

            <h2 class="title_header">Research articles and book chapters</h2>

            * = equal contributions (first-author)<br>
            ^ = equal contributions (second-author)<br>
            # = co-corresponding authors<br>
            
            <br>
            <h5 class="title_header"><a href="Doc/Publications_with_contribution.pdf" target="_blank">You can see here my contribution on each of the following papers</a></h5>	<br>  <!-- https://docs.google.com/document/d/1d3PWFX1YOcpqIpevIA0ea8KhB9YPwiBU37vfYonhKpk/edit -->
            <h5 class="title_header"><a href="https://scholar.google.fr/citations?user=pcevKk0AAAAJ&hl=en" target="_blank">Citations</a></h5>	<br>
            <h5 class="title_header"><a href="https://www.webofscience.com/wos/author/record/P-6297-2019" target="_blank">Profile as reviewer </a></h5> I have peer-reviewed articles for these journals: Bioinformatics, Genetics, Molecular Plant, Nucleic Acids Research,
Genome biology, Genome Biology and Evolution, Frontiers in Genetics, eLife. <br>
            
<!-- 
            <h5 class="title_header">Preprints:</h5> <br>
            
            
            
            <table>
              
              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">1. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://www.biorxiv.org/content/10.1101/2023.04.28.538684v1" target="_blank"><i> Identification of transcription factor co-binding patterns with non-negative matrix factorization. </i></a><br>
                  Rauluseviciute I, Launay T, Barzaghi G, Nikumbh S, Lenhard B, Krebs AR, <u>Castro-Mondragon JA</u>, and Mathelier A (2023) bioRxiv.</td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://www.biorxiv.org/content/10.1101/2023.04.28.538684v1" target="_blank"><img class="paper_img" src="images/Paper_21.png" /></a></td>
              </tr> 
            
            </table>

-->            
            <br>
            <br>
            <br>
            <h5 class="title_header">Peer-reviewed publications:</h5>
            <br>
            
            <table>


              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">23. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://journals.sagepub.com/doi/full/10.1177/11779322241272395" target="_blank"><i> RhizoBindingSites v2.0 Is a Bioinformatic Database of DNA Motifs Potentially Involved in Transcriptional Regulation Deduced From Their Genomic Sites </i></a><br>
                  Taboada-Castro H, AJ Hern√°ndez-√Ålvarez, <u>Castro-Mondragon JA</u>, and Encarnaci√≥n-Guevara S (2024) Bioinformatics and Biology Insights.</td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://journals.sagepub.com/doi/full/10.1177/11779322241272395" target="_blank"><img class="paper_img" src="images/Paper_23.jpeg" /></a></td>
              </tr> 

              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">22. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://doi.org/10.1093/nar/gkae743" target="_blank"><i> Identification of transcription factor co-binding patterns with non-negative matrix factorization. </i></a><br>
                  Rauluseviciute I, Launay T, Barzaghi G, Nikumbh S, Lenhard B, Krebs AR, <u>Castro-Mondragon JA</u>, and Mathelier A (2024) Nucleic Acids Research.</td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://doi.org/10.1093/nar/gkae743" target="_blank"><img class="paper_img" src="images/Paper_21.png" /></a></td>
              </tr> 
              
              

              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">21. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://doi.org/10.1093/nar/gkad1059" target="_blank"><i> JASPAR 2024: 20th anniversary of the open-access database of transcription factor binding profiles. </i></a><br>
                  Rauluseviciute I*, Riudavets-Puig R*, Blanc-Mathieu R^, <u>Castro-Mondragon JA</u>^, Ferenc K^, Kumar V^, Berhanu Lemma R^, Lucas J^, Cheneby J, Baranasic D, Khan A, Fornes O, Gundersen S, Johansen M, Hovig E, Lenhard B#, Sandelin A#, Wasserman WW#, Parcy F#, and Mathelier A#. (2024) Nucleic Acids Research.</td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://doi.org/10.1093/nar/gkad1059" target="_blank"><img class="paper_img" src="images/Paper_22.png" /></a></td>
              </tr> 

              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">20. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://doi.org/10.1093/nar/gkac1143" target="_blank"><i> Cis-regulatory mutations associate with transcriptional and post-transcriptional deregulation of gene regulatory programs in cancers</i></a><br>
                 <u>Castro-Mondragon JA</u>, Ragle Aure M, Lingj√¶rde OC, Langer√∏d A, Martens JWM, B√∏rresen-Dale AL, Kristensen VN, and Mathelier A (2022) Nucleic Acids Research.</td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://doi.org/10.1093/nar/gkac1143" target="_blank"><img class="paper_img" src="images/Paper_14.png" /></a></td>
              </tr> 
              
              
              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">19. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://elifesciences.org/articles/82459" target="_blank"><i> Genetic architecture of natural variations of cardiac performance in flies </i></a><br>
                 Saha S*, Spinelli L*, <u>Castro-Mondragon JA</u>, Kervadec A, Lynott M, Kremmer L, Roder L, Sallouha J, Torres M, Brun C, Vogler G, Bodmer R, Colas AR#, Ocorr K#, and Perrin L# (2021) eLife.</td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://elifesciences.org/articles/82459" target="_blank"><img class="paper_img" src="images/Paper_17.png"/></a></td>
              </tr> 


              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">18.</td>
                <td valign=top style="width:750px;height:auto;"> <a href="https://doi.org/10.1093/nar/gkac312" target="_blank"><i>RSAT 2022: regulatory sequence analysis tools</i></a><br>
                  Santana-Garcia W*, <u>Castro-Mondragon JA</u>*, Padilla-Galvez M, Nguyen NTT, Elizondo-Salas A, Ksouri N, Gerbes F, Thieffry D, Vincens P, Contreras-Moreira B#, van Helden J#, Thomas-Chollier M#, Medina-Rivera A#. (2022) Nucleic Acids Research </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:29px"><a href="https://doi.org/10.1093/nar/gkac312" target="_blank"><img class="paper_img" src="images/Paper_20.jpeg" /></a></td>
              </tr> 


                <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">17. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://doi.org/10.1093/nar/gkab1113" target="_blank"><i>JASPAR 2022: the 9th release of the open-access database of transcription factor binding profiles</i></a><br>
                 <u>Castro-Mondragon JA</u>*, Riudavets-Puig R*, Rauluseviciute I*, Berhanu Lemma R, Turchi L, Blanc-Mathieu R, Lucas J, Boddie P, Khan A, Manosalva Perez N, Fornes O, Leung TY, Aguirre A, Hammal F, Schmelter D, Baranasic D, Ballester B, Sandelin A#, Lenhard B#, Vandepoele K, Wasserman WW#, Parcy F#, and Mathelier A#. (2021) Nucleic Acids Research </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://doi.org/10.1093/nar/gkab1113" target="_blank"><img class="paper_img" src="images/Paper_19.png"/></a></td>
              </tr>     
              
                <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">16. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://www.nature.com/articles/s41467-021-26861-0" target="_blank"><i>  Epromoters function as a hub to recruit key transcription factors required for the inflammatory response</i></a><br>
                Santiago-Algarra D, Souaid C, Singh H, Dao TML, Hussain S, Medina-Rivera A, Ramirez-Navarro L, <u>Castro-Mondragon JA</u>, Sadouni N, Charbonnier G, Spicuglia S. (2021) Nature Communications </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://www.nature.com/articles/s41467-021-26861-0" target="_blank"><img class="paper_img" src="images/Paper_18.png"/></a></td>
              </tr>               
              
                <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">15. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-021-07760-6" target="_blank"><i>  UniBind: maps of high-confidence direct TF-DNA interactions across nine species</i></a><br>
                 Riudavets Puig R, Boddie P, Khan A, <u>Castro-Mondragon JA</u>, and Mathelier A (2021) BMC Genomics.</td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-021-07760-6" target="_blank"><img class="paper_img" src="images/Paper_16.png"/></a></td>
              </tr> 
                
                
                <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">14. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-021-00880-4" target="_blank"><i> Crosstalk between microRNA expression and DNA methylation drives the hormone-dependent phenotype of breast cancer </i></a><br>
                 Ragle Aure M,  Fleischer T, Bj√∏rklund S, Ankill J, <u>Castro-Mondragon JA</u>, OSBREAC (Oslo Breast Cancer Research Consortium), B√∏rresen-Dale AL, Tost J, Sahlberg KK,  Mathelier A, Tekpli X#, Kristensen VN# (2021) Genome Medicine.</td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-021-00880-4" target="_blank"><img class="paper_img" src="images/Paper_13.png" /></a></td>
              </tr> 
              
              
              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">13. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://doi.org/10.1093/plphys/kiaa091" target="_blank"><i> Tuning promoter boundaries improves regulatory motif discovery in non-model plants: the peach example</i></a><br>
                Ksouri N, <u>Castro-Mondragon JA</u>, Montardit-Tarda F, van Helden J, Contreras-Moreira B#, and Gogorcena Y# (2021) Plant Physiology.</td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://doi.org/10.1093/plphys/kiaa091" target="_blank"><img class="paper_img" src="images/Paper_12.png" /></a></td>
              </tr> 
              
              
              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">12. </td>
                <td valign=top style="width:750px;height:auto;"> <a href="https://doi.org/10.3389/fmicb.2020.567471" target="_blank"><i> RhizoBindingSites, a Database of DNA-Binding Motifs in Nitrogen-Fixing Bacteria Inferred Using a Footprint Discovery Approach </i></a><br>
                Taboada-Castro H, <u>Castro-Mondragon JA</u>, Aguilar-Vera A, Hernandez-Alvarez AJ, van Helden J, and Encarnacion-Guervara S. (2020) Frontiers in Microbiology </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:25px"><a href="https://doi.org/10.3389/fmicb.2020.567471" target="_blank"><img class="paper_img" src="images/Paper_15.jpg" /></a></td>
              </tr> 


              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">11. </td>
                <td valign=top style="width:750px;height:auto;"> <a href="https://doi.org/10.1093/nar/gkz1001" target="_blank"><i> JASPAR 2020: update of the open-access database of transcription factor binding profiles</i></a><br>
                Fornes O*, <u>Castro-Mondragon JA*</u>, Khan A*, van der Lee Robin, Zhang X, Richmond PA, Modi BP, Correard S, Gheorghe M, Baranasic D,  Santana-Garcia Walter, Tan G,  Cheneby J, Ballester B, Parcy F, Sandelin A#, Lenhard B#, Wasserman WW#, and Mathelier A#. (2019) Nucleic Acids Research </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:25px"><a href="https://doi.org/10.1093/nar/gkz1001" target="_blank"><img class="paper_img" src="images/Paper_11.png" /></a></td>
              </tr> 
              
              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">10.</td>
                <td valign=top style="width:750px;height:auto;"> <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky317/4990780" target="_blank"><i>RSAT 2018: regulatory sequence analysis tools 20th anniversary</i></a><br>
                Nguyen NTT*, Contreras-Moreira B*, <u>Castro-Mondragon JA</u>, Santana-Garcia W, Ossio R, Robles-Espinoza CD, Bahin M, Collombet S, Vincens P, Thieffry D, van Helden J#, Medina-Rivera A#, Thomas-Chollier M#. (2018) Nucleic Acids Research </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:29px"><a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky317/4990780" target="_blank"><img class="paper_img" src="images/Paper_10.png" /></a></td>
              </tr> 
              
              
              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">9. </td>
                <td valign=top style="width:700px;height:auto;"> <a href="https://doi.org/10.1099/mic.0.000720" target="_blank"><i> Proteins in the periplasmic space and outer membrane vesicles of Rhizobium etli CE3 grown in minimal medium are largely distinct and change with growth phase </i></a><br>
                Taboada H*, Meneses N*, Dunn MF*, Vargas-Lagunas C, Buchs N, <u>Castro-Mondragon JA</u>, Heller M, and Encarnacion S. (2018) Microbiology.</td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:80px"><a href="https://doi.org/10.1099/mic.0.000720" target="_blank"><img class="paper_img" src="images/Paper_09.png" /></a></td>
              </tr> 
              
              
              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">8.</td>
                <td valign=top style="width:750px;height:auto;"> <a href="https://academic.oup.com/nar/article/doi/10.1093/nar/gkx1126/4621338" target="_blank"><i>JASPAR 2018: update of the open-access database of transcription factor binding profiles and its web framework</i></a><br>
                Khan A*, Fornes O*, Stigliani A*, Gheorghe M, <u>Castro-Mondragon JA</u>, van der Lee R, Bessy A, Ch√®neby J, Kulkarni S, Tan G, Baranasic D, Arenillas D, Sandelin A#, Vandepoele K, Lenhard B#, Ballester B, Wasserman W#, Parcy F, Mathelier A#. (2017) Nucleic Acids Research </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:29px"><a href="https://academic.oup.com/nar/article/doi/10.1093/nar/gkx1126/4621338" target="_blank"><img class="paper_img" src="images/Paper_08.png" /></a></td>
              </tr>     
              
              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">7.</td>
                <td valign=top style="width:750px;height:auto;"> <a href="https://www.nature.com/ng/journal/v49/n7/full/ng.3884.html" target="_blank"><i>Genome-wide characterization of mammalian promoters with distal enhancer functions</i></a><br>
                Dao LM*, Galindo-Albarran AO*, <u>Castro-Mondragon JA^</u>, Andireu-Soler C^, Medina-Rivera A^, Souadi C, Charbonnier G, Griffon A, Vanhille L, Stephen S, Alomairi J, Soler C, Stephen T,  Martin D, Torres M, Fernandez N, Soler E, van Helden J, Puthier D, Spicuglia S (2017) Nature Genetics </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:29px"><a href="https://www.nature.com/ng/journal/v49/n7/full/ng.3884.html" target="_blank"><img class="paper_img" src="images/Paper_07.png" /></a></td>
              </tr>              

              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">6.</td>
                <td valign=top style="width:750px;height:auto;"> <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx314" target="_blank"><i>RSAT matrix-clustering: dynamic exploration and redundancy reduction of transcription factor binding motif collections</i></a><br>
                <u>Castro-Mondragon JA</u>, Jaeger S, Thieffry D, Thomas-Chollier M#, and van Helden J# (2017) Nucleic Acids Research </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:29px"><a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx314" target="_blank"><img class="paper_img" src="images/Paper_06.jpg" /></a></td>
              </tr>              
              
              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top;">
                <td valign=top style="width:15px;height:auto;">5.</td>
                <td valign=top style="width:750px;height:auto;"> <a href="http://link.springer.com/protocol/10.1007/978-1-4939-6396-6_19" target="_blank"><i>RSAT::Plants: Motif Discovery in ChIP-Seq Peaks of Plant Genomes</i></a><br>
                <u>Castro-Mondragon JA</u>*, Rioualen C*, Contreras-Moreira B, van Helden J (2016) Plant Synthetic Promoters - Springer Protocol </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:29px"><a href="http://link.springer.com/protocol/10.1007/978-1-4939-6396-6_19" target="_blank"><img class="paper_img" src="images/Paper_05.jpg" /></a></td>
              </tr>
              
              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top">
                <td valign=top style="width:15px;height:auto;">4.</td>
                <td valign=top style="width:750px;height:auto;"> <a href="http://link.springer.com/protocol/10.1007/978-1-4939-6396-6_18" target="_blank"><i>RSAT::Plants: Motif Discovery Within Clusters of Upstream Sequences in Plant Genomes</i></a><br>
                Contreras-Moreira B#, <u>Castro-Mondragon JA</u>, Rioualen C, Cantalapiedra CP, van Helden J (2016) Plant Synthetic Promoters - Springer Protocol </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:29px"><a href="http://link.springer.com/protocol/10.1007/978-1-4939-6396-6_18" target="_blank"><img class="paper_img" src="images/Paper_04.jpg" /></a></td>
              </tr>

              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top">
                <td valign=top style="width:15px;height:auto;">3.</td>
                <td valign=top style="width:750px;height:auto;"> <a href="http://nar.oxfordjournals.org/content/44/D1/D133.long"><i>RegulonDB version 9.0: high-level integration of gene regulation, coexpression, motif clustering and beyond</i></a><br>
                Gama-Castro S*, Salgado H*, Santos-Zavaleta A, Ledezma-Tejeida D, Mu√±iz-Rascado L, Garc√≠a-Sotelo JS, Alquicira-Hern√°ndez K, Mart√≠nez-Flores I, Pannier L, <u>Castro-Mondrag√≥n JA</u>, Medina-Rivera A, Solano-Lira H, Bonavides-Mart√≠nez C, P√©rez-Rueda E, Alquicira-Hern√°ndez S, Porr√≥n-Sotelo L, L√≥pez-Fuentes A, Hern√°ndez-Koutoucheva A, Del Moral-Ch√°vez V, Rinaldi F, Collado-Vides J. (2015) Nucleic Acids Research </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:29px"><a href="http://nar.oxfordjournals.org/content/44/D1/D133.long" target="_blank"><img class="paper_img" src="images/Paper_03.png" /></a></td>
              </tr>              

              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top">
                <td valign=top style="width:15px;height:auto;">2.</td>
                <td valign=top style="width:750px;height:auto;"> <a href="http://nar.oxfordjournals.org/content/43/W1/W50"><i>RSAT 2015 : Regulatory Sequence Analysis Tools</i></a><br>
                Medina-Rivera A*, Defrance M*, Sand O*, Herrmann C, <u>Castro-Mondragon JA</u>, Delerce J, Jaeger S, Blanchet C, Vincens P, Caron C, Staines DM, Contreras-Moreira B, Artufel M, Charbonnier‚ÄìKhamvongsa L, Hernandez C, Thieffry D, Thomas-Chollier M#, van Helden J# (2015) Nucleic Acids Research </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:29px"><a href="http://nar.oxfordjournals.org/content/43/W1/W50" target="_blank"><img class="paper_img" src="images/Paper_02.png" /></a></td>
              </tr>    

              <tr colspan=3 valign=top align=left style="font-size:15px;display:inline-table;vertical-align:top">
                <td valign=top style="width:15px;height:auto;">1.</td>
                <td valign=top style="width:750px;height:auto;"> <a href="http://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-575"><i>Genomic basis of symbiovar mimosae in Rhizobium etli</i></a><br>
                Rogel MA, Bustos P, Santamar√≠a RI, Gonz√°lez V, Romero D, Miguel AC, Lozano L, <u>Castro-Mondragon JA</u>, Mart√≠nez-Romero J, Orme√±o-Orrillo E, Mart√≠nez-Romero E. (2014) BMC Genomics </td>
					       <td valign=top style="width:auto;height:auto;display:inline-table;vertical-align:top;margin-left:29px"><a href="http://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-575" target="_blank"><img class="paper_img" src="images/Paper_01.png" /></a></td>
              </tr> 
              
            </table>
            
            
            
            

            </div>


      
      <div class="twelve columns header-col">
               
              <br>
              <h5 class="title_header">Posters presented in conferences</h5>
					<ul>
					  
										<li><u>Castro-Mondragon JA</u>, Ragle Aure M, Kristensen V,  Mathelier A<br/><i> "Combining transcriptional and post-transcriptional regulation to predict mutations altering the gene regulatory program in cancer cells"</i>. <br/> Systems Biology; Global Regulation of Gene Expression, CSHL, USA. <a href="Doc/Posters/SYSTEMS_20_Castro-Mondragon.pdf" target="_blank">[Full text]</a></li>
				  
					<li><u>Castro-Mondragon JA</u>, Ragle Aure M, Kristensen V,  Mathelier A<br/><i> "Combining transcriptional and post-transcriptional regulation to predict mutations altering the gene regulatory program in cancer cells"</i>. <br/> ISMB/ECCB 2019, Basel, Switzerland. <a href="https://f1000research.com/posters/8-1286" target="_blank">[Full text]</a></li>

					<li><u>Castro-Mondragon JA</u>, Thieffry D, Thomas-Chollier M,  van Helden J<br/><i> "Clustering and redundancy reduction of transcription factor binding motifs"</i>. <br/> European Conference on Computational Biology (ECCB16), The Hague, Netherlands. <a href="http://f1000research.com/posters/5-2174" target="_blank">[Full text]</a></li>
					
					<li><u>Castro-Mondragon JA</u>, Thomas-Chollier M,  van Helden J<br/><i> "Comparing   and   clustering   multiple   collections   of   DNA   motifs   using   RSAT"</i>. <br/> Basel Computational Biology Conference (BC2 2015), Basel, Switzerland. <a href="http://f1000research.com/posters/5-297" target="_blank">[Full text]</a></li>
					
					<li><u>Castro-Mondragon JA</u>, Thomas-Chollier M, Thieffry D,  van Helden J<br/><i> "Comparing,   clustering   and   aligning   Transcription   Factor   Binding   Motifs   with   RSAT"</i>. <br/> European Conference on Computational Biology (ECCB14), Strasbourg, France. <a href="http://f1000research.com/posters/1097149" target="_blank">[Full text]</a></li>					

					</ul>
            </div>

      </div><!-- Row End-->

   </section> <!-- Contact Section End-->


     <!-- Contact Section
   ================================================== -->
  <section id="parallel_dimensions">
    <h2><a href="#parallel_dimensions">.</a></h2>
    
      <div class="row">

        <div class="ten columns header-col">

          <p><h3>Parallel dimensions</h3></p>
          
          <ol>
            
            
            <li> A brief interview about my time as researcher at <a href="https://www.med.uio.no/ncmm/english/" target="_blank">NCMBM</a>, a summary of my scientific career (until 2022), and my <a href="https://projects.au.dk/nordic-embl-partnership/show-news/artikel/a-career-in-bioinformatics-from-academia-to-biotech" target="_blank">transition from academia to industry</a>. Although I am back to the public sector since May 2025, so I can give you my opinion in the other direction.
            </li><br/>
            
            
            <li> Once upon a time, I had a free day in <a href="https://www.youtube.com/watch?v=s4ftEdW2wdo" target="_blank">NYC</a> and visited the <a href="https://www.moma.org/" target="_blank">MoMA</a>. Among the pieces there I was curious about the <a a href="https://www.moma.org/collection/works/205997" target="_blank">'Ajedrez Visual (Visual Chess)'</a> by Antonieta Sosa. I tried to recreate this piece through my very first Shiny app: <a href="https://jaimicore.shinyapps.io/visualchess/" target="_blank">Random Visual Chess</a>.
            </li><br/>
            

<!--            
            <li> I am a core member of the <a href="https://rsg-norway.iscbsc.org/" target="_blank">ISCB Regional Student Group (RSG) Norway</a>, a non-profit organisation aimed at creating a platform that will have a crucial role in fostering interactions and networking among students, early-career researchers and professionals in computational biology and related fields.
            </li>
-->            
            
            <li> Among my personal non-academic interest is understanding the arithmetic of mayans to measure time, for example the vigesimal system, how they measured time using multiple calendars related to different natural cycles, and how these calendars are synchronized and used nowadays. 
            </li><br/>

          </ol>
          
    	  <!--<ul> <a href="Jiawei_debug_archive.zip" target="_blank">Jiawei</a> output
          </ul>
          -->
	  
        </div>
      </div>
      
      <br>      <br>      <br>      <br>      <br>      <br>      <br>      <br>      <br>      <br>
    

  </section> <!-- Contact Section End-->

  <!-- footer
  ================================================== -->
  <footer>
    
    <div class="row">
      
      <div class="twelve columns">
	
        <ul class="copyright">
<!--           <li>&copy; Copyright 2014 CeeVee</li>
-->
          <li>Design by <a title="Styleshout" href="http://www.styleshout.com/">Styleshout</a></li>   
        </ul>
	
      </div>
      
      <div id="go-top" style="display: block;"><a class="smoothscroll" title="Back to Top" href="#home"><i class="icon-up-open"></i></a></div>
      
    </div>
    
  </footer> <!-- Footer End-->
  
  <!-- Java Script
  ================================================== -->
  <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
  <script>window.jQuery || document.write('<script src="js/jquery-1.10.2.min.js"><\/script>')</script>
    <script type="text/javascript" src="js/jquery-migrate-1.2.1.min.js"></script>
    
    <script src="js/jquery.flexslider.js"></script>
    <script src="js/waypoints.js"></script>
    <script src="js/jquery.fittext.js"></script>
    <script src="js/magnific-popup.js"></script>
    <script src="js/doubletaptogo.js"></script>
    <script src="js/spin.js"></script>
    
    <script src="js/init.js"></script>
    
</body>

</html>